tests/Oryx.Integration.Tests/Node/NodeTestUsingZippedNodeModules.cs

```diff
@@ -23,8 +23,11 @@ public NodeTestUsingZippedNodeModules(ITestOutputHelper output, TestTempDirTestF
         }
 
         [Theory]
-        [MemberData(nameof(TestValueGenerator.GetZipOptions_NodeVersions), MemberType = typeof(TestValueGenerator))]
-        public async Task CanBuildAndRunNodeApp_UsingZippedNodeModules(string compressFormat, string nodeVersion)
+        //[MemberData(nameof(TestValueGenerator.GetZipOptions_NodeVersions), MemberType = typeof(TestValueGenerator))]
```

Comment:
this test helps to make sure we have the compression tools in every image. I'm fine with removing it as long as we have a test that uses compressed tar-gz node_modules for every node version.